src/qml/patchmanager.cpp

```diff
@@ -115,9 +115,7 @@ void PatchManager::restartServices()
     }
 
     if (m_homescreenNeedRestart) {
-        QStringList arguments;
-        arguments << "--user" << "restart" << "lipstick.service";
-        QProcess::startDetached("systemctl", arguments);
+        restartLipstick();
```

Comment:
Is there a DBus call to restart lipstick ? Or do you want to go with a call to systemd ?